cmake/ExternalPalace.cmake

```diff
@@ -6,7 +6,16 @@
 #
 
 # Force build order
-set(PALACE_DEPENDENCIES mfem libCEED)
+if(PALACE_WITH_SPACK)
+  # We want our external libCEED
+  # TODO: Add support for external mfem
+  # - https://github.com/mfem/mfem/issues/4792
+  # - Need to add patches
+  # - Need to support shared / static or both
+  set(PALACE_DEPENDENCIES mfem)
```

Comment:
You're almost able to just remove this and have palace use spack installed mfem, if not for the lack of CMake build support in mfem's package.py.